workspaces/ui-v2/src/store/selectors/visibilitySelectors.ts

```diff
@@ -0,0 +1,81 @@
+import { ChangeType } from '<src>/types';
+
+export const isItemVisible = <T extends { isRemoved: boolean }>(
```

Comment:
These are generic functions we can hopefully use for shape visibility (and any other removable items)

Reply:
Those kinds of selectors will definitely come in handy for the shape editing stuff as well. Good that it's isolated, makes it easier to cover with tests later as well.